src/elm/Page/Community/Settings/Features.elm

```diff
@@ -255,6 +271,20 @@ saveFeature feature state authorization accountName community =
                 Objectives ->
                     state
 
+                Kyc ->
+                    community.hasObjectives
+
+        hasKyc =
+            case feature of
+                Shop ->
+                    community.hasKyc
+
+                Objectives ->
+                    community.hasKyc
+
+                Kyc ->
+                    state
```

Comment:
@victorgdev can you, please, change these three bindings to be like this:
```elm
hasShop =
    case feature of
        Shop ->
            state

        _ ->
            community.hasShop

hasObjectives =
    case feature of
        Objectives ->
            state

        _ ->
            community.hasObjectives

hasKyc =
    case feature of
        Kyc ->
            state

        _ ->
            community.hasKyc
```
I think this way of doing pattern matching represents the purpose of the code better. @RodrigoLeiteF are you ok with this change?

Reply:
Agreed. No idea why I didn't think of that originally :p

Reply:
I will resolve this today!!